clusterloader2/cmd/clusterloader.go

```diff
@@ -61,7 +60,8 @@ var (
 )
 
 func initClusterFlags() {
-	flags.StringEnvVar(&clusterLoaderConfig.ClusterConfig.KubeConfigPath, "kubeconfig", "KUBECONFIG", "", "Path to the kubeconfig file")
+	flags.StringEnvVar(&clusterLoaderConfig.ClusterConfig.KubeConfigPath, "kubeconfig", "KUBECONFIG", "", "Path to the kubeconfig file, --run-from-cluster must be false")
+	flags.BoolEnvVar(&clusterLoaderConfig.ClusterConfig.RunFromCluster, "run-from-cluster", "RUN_FROM_CLUSTER", false, "creates using in-cluster client config, --kubeconfig must be unset")
```

Comment:
nit: the description text seems strange (some noun is missing; I guess you meant

`Whether to use in-cluster client-config to create a client, --kubeconfig must be unset`